src/interaction/InteractionManager.js

```diff
@@ -414,7 +412,7 @@ InteractionManager.prototype.onMouseDown = function (event)
 {
     this.mouse.originalEvent = event;
     this.eventData.data = this.mouse;
-    this.eventData.stopped = false;
+    this.eventData.stopped = event.defaultPrevented;
```

Comment:
What is this supposed to do? `preventDefault()` is not the same as `stopPropagation()` (which sets `.stopped`).